Apps/WebClient/src/ClientApp/test/router.test.ts

```diff
@@ -1,10 +1,12 @@
+// eslint-disable-next-line
+import container from "@/plugins/inversify.container";
```

Comment:
Why does this needed to be imported first?  And why does reports.test.ts import `inversify.config` first instead?

Reply:
Inversify (DI library) needs to be imported pretty early before other definitions come into play to make sure the dependency injection works properly. This is also why reflect-metadata needs to be imported early on and only once